docs/source/history.rst

```diff
@@ -5,6 +5,44 @@ Release history
 
 .. towncrier release notes start
 
+pytest-trio 0.8.0 (2022-11-01)
+------------------------------
+
+Features
+~~~~~~~~
+
+- If a test raises an ``ExceptionGroup`` (or nested ``ExceptionGroup``\ s) with only
```

Comment:
Does this definitely render right? I would write `\s` with no space here, and skip the `\ ` before a comma, but my sphinx-fu is not elevated enough to know whether these will actually be parsed differently.

Reply:
It does render right: https://pytest-trio--131.org.readthedocs.build/en/131/history.html

Well, https://pytest-trio.readthedocs.io/en/stable/history.html is up now already.